third_party/move/move-compiler-v2/src/lib.rs

```diff
@@ -34,6 +34,10 @@ use move_stackless_bytecode::function_target_pipeline::{
 };
 use move_symbol_pool::Symbol;
 pub use options::*;
+use pipeline::{
```

Comment:
The import line is added by VS Code. It is acceptable because we defined the `pipeline` module on line 11. Though it may be more uniform to merge this import into the above `use crate::pipeline` or reversely.